tasks/db_deploy/install/orca_sql.py

```diff
@@ -1043,7 +1043,5 @@ def reconcile_phantom_report_table_sql() -> text:  # pragma: no cover
               IS 'Last update of the object as reported in the ORCA catalog.';
             COMMENT ON COLUMN reconcile_phantom_report.orca_size
               IS 'Size in bytes of the object as reported in the ORCA catalog.';
-            COMMENT ON COLUMN reconcile_phantom_report.s3_storage_class_id
-              IS 'Storage class of the file as reported in the ORCA catalog.';
```

Comment:
do we have to remove the line 1019?
```
, orca_storage_class_id int2 NOT NULL
```

Reply:
No, that one is still in use.

Reply:
Because the column does not exist

Reply:
oh I see. I just noticed s3_storage_class_id and orca_storage_class_id are different terms.